alcs-frontend/src/app/features/application/post-decision/post-decision.component.html

```diff
@@ -131,7 +131,7 @@ <h5 id="modifications" class="post-decision-header">
             (save)="onSaveModificationOutcome(modification.uuid, $event)"
           ></app-inline-review-outcome>
         </span>
-        <span *ngIf="modification.reviewOutcome.code !== 'PEN'">{{ modification.reviewOutcome?.label }}</span>
+        <span *ngIf="modification.reviewOutcome.code !== 'PEN'">{{ modification.reviewOutcome.label }}</span>
```

Comment:
So now it properly handles undefined?

Reply:
I think its more that now the linter suggests places where you used ? and dont actually need it